example-cucumber/src/test/java/com/epam/reportportal/example/cucumber/logging/ReportsTestWithParameters.java

```diff
@@ -2,20 +2,27 @@
 
 import cucumber.api.java.en.Given;
 import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class ReportsTestWithParameters {
 
-	private static final Logger LOGGER = LoggerFactory.getLogger(ReportAttachmentsTest.class);
+	private static final Logger LOGGER = LoggerFactory.getLogger(ReportsTestWithParameters.class);
 
 	@Given("It is test with parameters")
 	public void infoLevel() {
 		LOGGER.info("It is test with parameters");
 	}
 
+	@When("^I have (.*) parameter$")
```

Comment:
Please also use this type of parameter notation: `I have <int> <string> in my pocket` (6, apples). Adding another step with both of them should be OK.

Reply:
added